ansible/tasks/logging/install_fluentd.yaml

```diff
@@ -0,0 +1,24 @@
+---
+- name: Create temporary directory on windows host
+  win_tempfile:
+    state: directory
+  register: tempdir_win
+- name: Fetch ruby devkit exe
+  win_get_url:
+    url: "{{ ruby_fetch_url }}"
+    dest: "{{ tempdir_win.path }}\\ruby_devkit.exe"
+- name: Run ruby devkit exe
```

Comment:
I see the `when` clause being used in #21. Shouldn't that be used here too? I would like to have consistency regarding such things across our playbooks.

Reply:
I'm not sure we would need it here since the task will not be executed if the tempfile creation failed.